starfish/image/_stack.py

```diff
@@ -38,8 +38,8 @@ def __init__(self, image_partition):
             self.set_slice(indices={Indices.HYB: h, Indices.CH: c, Indices.Z: zlayer}, data=tile.numpy_array)
 
     @classmethod
-    def from_image_stack(cls, image_stack_name_or_url, baseurl):
-        image_partition = Reader.parse_doc(image_stack_name_or_url, baseurl)
+    def from_url(cls, relativeurl, baseurl):
+        image_partition = Reader.parse_doc(relativeurl, baseurl)
 
         return ImageStack(image_partition)
```

Comment:
I usually see `return cls(image_partition)` here instead of calling the object's formal name (else, why have an `@classmethod` tag to expose `cls` instead of an `@staticmethod`?)

Reply:
will fix.